### PR TITLE
Adding squirrel-core recipe

### PR DESCRIPTION
**Pull request by ThomasWollmann**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ThomasWollmann** [comment]
https://github.com/orgs/conda-forge/teams/help-python This pull request is ready for approval.

File: recipes/squirrel-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{% set name = "squirrel-core" %}
+{% set version = "0.12.0.dev8123371" %}
```
Comment (ocefpaf) [inline recipes/squirrel-core/meta.yaml:2]: This is a dev release and you would need a local config to send it to a dev channel. See https://github.com/conda-forge/cfep/blob/main/cfep-05.md for more info.

Is there a stable release you want to get first on the channel and work on the dev release later in the feedstock?

Comment (ThomasWollmann) [reply recipes/squirrel-core/meta.yaml:2]: Soon, the first non dev release will be published. Will upgrade the yaml before merging.

Comment (bgruening) [reply recipes/squirrel-core/meta.yaml:2]: Sounds good 😊

Comment (ocefpaf) [reply recipes/squirrel-core/meta.yaml:2]: Please ping us when you do so we can merge,

**Comment by ThomasWollmann** [comment]
ping @ocefpaf @bgruening

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**